src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -468,4 +471,64 @@ object Task {
   }
 
   implicit val taskFormat: Format[Task] = Json.format[Task]
+
+  def provisioned(taskId: Id, networkInfo: NetworkInfo, appVersion: Timestamp, now: Timestamp): Task = {
+    Task(
+      taskId = taskId,
+      runSpecVersion = appVersion,
+      status = Task.Status(stagedAt = now, condition = Condition.Provisioned, networkInfo = networkInfo)
+    )
+  }
+}
+
+object Tasks {
+  def provisioned(taskIds: Seq[Id], agentInfo: Instance.AgentInfo, hostPorts: Seq[Option[Int]], pod: PodDefinition, now: Timestamp): Seq[Task] = {
```

Comment:
Hm, It seems strange that the `provisioned` methods are in different objects.

Reply:
I just wanted to have a plural in a name so `Tasks.provisioned` since it returns a collection...

Reply:
We can unify them in a follow up.